modules/ossm-rn-known-issues.adoc

```diff
@@ -198,6 +201,14 @@ New issues for Kiali should be created in the link:https://issues.redhat.com/pro
 
 These are the known issues in Kiali:
 
+* https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click the **Node graph** menu option of any node menu within the traffic graph, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. Currently, no workaround exists for this issue.
```

Comment:
This is specifically mentioning 4.15. Should this known issue only be present in the 4.15 branch then (instead of in the 4.12-4.14 branches too)?